src/inputs/Table/components/TableToolbar/components/RevertDataFromDatasetButton/RevertDataFromDatasetButton.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React from 'react';
+import PropTypes from 'prop-types';
+import RestorePageIcon from '@mui/icons-material/RestorePage';
+import { IconButton } from '@mui/material';
+import { FadingTooltip } from '../../../../../../misc';
+
+export const RevertDataFromDatasetButton = ({ disabled, onRevert, label }) => {
```

Comment:
Maybe we should rename the component from `RevertDataFromDatasetButton` to `RevertDataButton` (or `ResetButton`)? From the ui components point of view, this button is generic and can call any "revert" function, it could be from a dataset, but also from any other sources (azure function, HTTP call, hard-coded JS function, ...)

Reply:
good idea ! I'd like to stick to `RevertButton` to avoid any confusion between webapp and generic components :slightly_smiling_face: